keyboards/ferris/0.1/0.1.c

```diff
@@ -14,4 +14,4 @@ GNU General Public License for more details.
 You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
-#include "ferris.h"
+#include "0.1.h"
```

Comment:
This should be renamed `0_1`

Reply:
Done here: https://github.com/qmk/qmk_firmware/pull/10564/commits/a9a773f6bc4661286c8a361ac6d8abaca413d6dd